gitlab-pipeline/stage/release.yml

```diff
@@ -6,7 +6,7 @@
   stage: release
   image: docker:19.03
```

Comment:
I fear that might have to be aligned as well

Reply:
Merging, and testing the release job after